topics/variant-analysis/tutorials/MTB_transmission/tutorial.md

```diff
@@ -0,0 +1,812 @@
+> https://zenodo.org/record/6010176/files/ERR2704687.TBprof.txt
+> https://zenodo.org/record/6010176/files/ERR313115.TBprof.txt
+> https://zenodo.org/record/6010176/files/ERR551620.TBprof.txt
+> https://zenodo.org/record/6010176/files/ERR5987300.TBprof.txt
+> https://zenodo.org/record/6010176/files/ERR5987352.TBprof.txt
+> https://zenodo.org/record/6010176/files/ERR6362078.TBprof.txt
+> https://zenodo.org/record/6010176/files/ERR6362138.TBprof.txt
+> https://zenodo.org/record/6010176/files/ERR6362139.TBprof.txt
+> https://zenodo.org/record/6010176/files/ERR6362156.TBprof.txt
+> https://zenodo.org/record/6010176/files/ERR6362253.TBprof.txt
+> https://zenodo.org/record/6010176/files/ERR6362333.TBprof.txt
+> https://zenodo.org/record/6010176/files/ERR6362484.TBprof.txt
+> https://zenodo.org/record/6010176/files/ERR6362653.TBprof.txt
+> https://zenodo.org/record/6010176/files/SRR13046689.TBprof.txt
+> https://zenodo.org/record/6010176/files/SRR998584.TBprof.txt
+>    ```
+> Create a **Dataset List (Collection)** for all the report files.
+>
+> Use a meaningful name, for example **TBprofiler reports**.
+{: .hands_on}
+
+## Summarize the data
+TB-profiler reports are very useful and comprehensive, and we will use them to better investigate
+drug resistance in our dataset. However, it is always useful to summarize the data on a per-sample basis,
+on a table, so we can quickly check which strains are, for example, MDR, an which are pan-susceptible.
+We would like to generate a table like the following:
+
+| Sample                  | DR profile |
+|-------------------------|------------|
+| Sample A |    Sensitive      |
+| Sample B | Sensitive         |
+| Sample C | MDR        |
+| Sample Z | XDR        |
+
+If we have a look at a TB profiler report we can see that there is one line describing the genotypic drug susceptibility.
+Let's have a look at the first part of the TB-profiler report for sample ERR6362653:
+
+```
+
+TBProfiler report
+=================
+
+The following report has been generated by TBProfiler.
+
+Summary
+-------
+ID: tbprofiler
+Date: Fri Jan 28 13:14:47 2022
+Strain: lineage2.2.1
+Drug-resistance: MDR
+
+Lineage report
+--------------
+Lineage	Estimated Fraction	Family	Spoligotype	Rd
+lineage2	1.000	East-Asian	Beijing	RD105
+lineage2.2	0.996	East-Asian (Beijing)	Beijing-RD207	RD105;RD207
+lineage2.2.1	0.999	East-Asian (Beijing)	Beijing-RD181	RD105;RD207;RD181
+```
+
+As you can see, this strain is multi-drug resistant as inficated by (`Drug-resistance: MDR`)
+We could then look for this information in each TB-profiler report and generate this table manually,
+for example in a spreadsheet. However this is not feasible when analyzing hundreds or thousands of samples
+(and very **error-prone!**).
+
+We are here to learn bioinformatics, so let's generate this table using Linux commands.
+
+The process will consist on three steps:
+* 1) Select the line containing the drug resistance profile with **grep**:
```

Comment:
this will not be a proper numbered list, please use 
```suggestion
1. Select the line containing the drug resistance profile with **grep**:
```
and then check the indentation of the child elements below (generally three spaces to match the text 'Select')